### PR TITLE
Adding linterns to the feature branch 

### PR DESCRIPTION
**Pull request by JbirdL86**
In this pull request we are trying to fix the error flags on the linters.

### CODE REVIEW

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hello team 👋. Hands down this is the most detailed project I have ever seen. You guys gave me a hard time figuring out which was the original page. Perfect 💯 !

You made your clone in such a way that it cannot be touched 👌! Now would you please make a killer README, because right now it just contains the name of the project.

Apply this small change and request another code review.

Happy coding 👨‍💻

**Review by JbirdL86 — COMMENTED** [review]
Thank you very much for your positive feedback! We are very exited about your comments, it really makes us see our progress from 0 to a whole other level! Thank you!!!!

I think we are ready now @tirthajyoti-ghosh .

**Comment by JbirdL86** [comment]
@tirthajyoti-ghosh i'm not sure how to proceed to resubmit the pull request. Can you help me?

**Review by elmejdki — APPROVED** [review]
# status Approved :heavy_check_mark: 
## Great job guys your clone looks exactly as the original :100: 

now you can merge to master you project is approved :+1: 

## Happy Coding :tophat:

File: README.md [hunk 1]
```diff
@@ -1 +1,74 @@
...
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```
Comment (elmejdki) [inline README.md]: Please remove all of this empty titles

File: README.md [hunk 2]
```diff
@@ -1 +1,74 @@
-# Forms
\ No newline at end of file
+# HTML Forms
+
+> This one is a mint.com’s signup page clone. To make possible this project we went through a research on how to make HTLM forms, so Input tags were used to elaborate the user information request section. Using anchors and button tags, border styles, background colors, fixed and relative positions and cursor properties, helped us to make buttons. Icons are taken from fontawesome and directly from the images URLs with the aid of Chrome's extention Web Developer. For building labels and desciption texts, we worked mainly with font rules in CSS, like sizing, font family, weight, color, just to mention a few of them. Responsiveness was added to the more generic and main containers, while a tendency to fixed elements and block display can be found in containers inside these first ones. Elements alignment was made by setting margin and padding CSS rules. From the beginning, one of our principal focus was to order code lines and look for those repetitive rules that could be grouped into the less possible number of class selectors in CSS.
+
+![screenshot](./assets/images/formsScreenShot.png)
```
Comment (elmejdki) [inline README.md]: and fix the image is not showing up I proved your project cause you showed a great job but you need to fix this before you go to another project

**Review by elmejdki — APPROVED** [review]

**Comment by JbirdL86** [comment]
Thank you guys! We are exited about your comments!

**Pull request merged**